<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="1" column="0" colspan="4">
     <widget class="QTableWidget" name="tableWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>ReferenceNumber</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>PurchaseOrderNumber</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipCarrier</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipService</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipBilling</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipAccount</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipDate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CancelDate</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Notes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipTo Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToCompany</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToAddress1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToAddress2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToCity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToState</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToZip</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToCountry</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToPhone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToFax</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToEmail</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToCustomerID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ShipToDeptNumber</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RetailerID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SKU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Quantity</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UseCOD</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>UseInsurance</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Saved Elements</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Order Item Saved Elements</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Carrier Notes</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>555</width>
        <height>210</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>555</width>
        <height>210</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>555</width>
        <height>210</height>
       </size>
      </property>
      <property name="title">
       <string>Summary Information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0" columnminimumwidth="0,400">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sender:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>To:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Subject:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of drops:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Failures:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Amia Drops:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Standard Drops:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblSender">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTo">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSubject">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblNumDrops">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFailures">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblAmiaDrops">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblBaxDrops">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>210</height>
       </size>
      </property>
      <property name="title">
       <string>Failures</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>210</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Shipper</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtShipper">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Consignee</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtConsignee">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Pickup Location</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QTextEdit" name="txtPickupLocation">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnExport">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFIle"/>
  </widget>
  <action name="actionOpen">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>loadRecord(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>exportData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1046</x>
     <y>778</y>
    </hint>
    <hint type="destinationlabel">
     <x>935</x>
     <y>779</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadData()</slot>
  <slot>loadRecord(int,int)</slot>
  <slot>exportData()</slot>
 </slots>
</ui>
